python/src/aiconfig/editor/server/server.py

```diff
@@ -114,6 +117,61 @@ def create() -> FlaskResponse:
     state.aiconfig = AIConfigRuntime.create()  # type: ignore
     return HttpResponseWithAIConfig(message="Created new AIConfig", aiconfig=state.aiconfig).to_flask_format()
 
+@app.route('/api/test_streaming', methods=["POST"])
+def test_streaming():
+    EXCLUDE_OPTIONS = {
+        "prompt_index": True,
+        "file_path": True,
+        "callback_manager": True,
+    }
+    state = get_server_state(app)
+    request_json = request.get_json()
+    num_stream_steps : int = request_json.get("num_stream_steps", 10)
+    print(f"{num_stream_steps=}")
+    print(f"{type(num_stream_steps)=}")
+    def generate(num_stream_steps: int):
+        prompt : Prompt = state.aiconfig.get_prompt('get_activities')
+        for i in range(num_stream_steps):
+            time.sleep(1)
+            output = ExecuteResult(
+                output_type="execute_result",
+                execution_count=0,
+                data = "Rossdan" + str(i+1),
+                metadata = {},
+            )
+            prompt.outputs = [output]
+            print(f"Done step {i+1}/{num_stream_steps}...")
+
+            aiconfig_json = state.aiconfig.model_dump(exclude=EXCLUDE_OPTIONS)
+            # print(f"{aiconfig_json=}\n")
+            # yield_output = core_utils.JSONObject({"data": aiconfig_json})
+            # print(f"{yield_output=}")
+            print(f"{str(aiconfig_json)=}\n")
+            yield str(aiconfig_json) + "\n\n"
```

Comment:
How does this iterate over the tokens returned? Doesn't it just yield the same fixed contents of `get_activities()` every iteration ?

Reply:
The `aiconfig_json` is updated with different output each time. Sorry I forgot to include test video but you can see it now :)